server/service/scripts.go

```diff
@@ -382,8 +383,35 @@ func getScriptResultEndpoint(ctx context.Context, request interface{}, svc fleet
 		return getScriptResultResponse{Err: err}, nil
 	}
 
-	// TODO: move this logic out of the endpoint function and consolidate in either the service
-	// method or the fleet package
+	return setUpGetScriptResultResponse(scriptResult), nil
+}
+
+type getDeviceSoftwareUninstallResultsRequest struct {
+	Token       string `url:"token"`
+	ExecutionID string `url:"execution_id"`
+}
+
+func (r *getDeviceSoftwareUninstallResultsRequest) deviceAuthToken() string {
+	return r.Token
+}
+
+func getDeviceSoftwareUninstallResultsEndpoint(ctx context.Context, request interface{}, svc fleet.Service) (fleet.Errorer, error) {
```

Comment:
I noticed the `get_device_software_install_results` is handled in `server/service/software_installers.go`. I don't think putting this code into `software_installers.go` matters much, but the `GetSoftwareInstallResults` method that `getSoftwareInstallResultsEndpoint` relies on is under `ee` https://github.com/fleetdm/fleet/blob/main/ee/server/service/software_installers.go#L1376
Does it matter that this isn't under `ee`?

Reply:
Good catch. This is a weird endpoint where we're basically returning script results, but within the context of installs, and we're filtering down to just uninstall scripts, so this should be in EE && software installers. Will fix now.